.github/workflows/linting.yml

```diff
@@ -17,4 +17,5 @@ jobs:
       - name: Run linter
         uses: golangci/golangci-lint-action@v2
         with:
-          version: v1.45
+          version: v1.52
+          args: --timeout=2m
```

Comment:
Can we configure this in the yaml config instead?

Reply:
moved